src/client/resources/conversions.ts

```diff
@@ -440,6 +440,11 @@ export function updateMigPlanFromValues(
   if (planValues.planClosed) {
     updatedSpec.closed = true;
   }
+  if (currentPlan.status) {
```

Comment:
Every MigPlan update issued by the UI will now set MigPlan.spec.refresh = true. There is probably a better way to do this that I'd be happy to adopt.